### PR TITLE
Update pt_BR translation

### PR DESCRIPTION
**Pull request by ComicShrimp**
Some suggestions for the Portuguese translation.

### CODE REVIEW

**Comment by Morganamilo** [comment]
@DatCrazyDelphox @ojmarcelino keysym

**Comment by DatCrazyDelphox** [comment]
Most suggestions are fine, although I think the last one referring to "pacotes a excluir" should stay as is

Since it prevents some misunderstanding in part of the user

**Comment by ComicShrimp** [comment]
@DatCrazyDelphox that's true, I'll change back

**Review by DatCrazyDelphox — APPROVED** [review]

File: po/pt_BR.po [hunk 1]
```diff
@@ -192,7 +192,7 @@ msgstr "pacotes ausentes no AUR: "
 
 #: src/download.rs:94
 msgid "marked out of date: "
-msgstr "desatualizados: "
+msgstr "marcado como desatualizado: "
 
 #: src/download.rs:105
 msgid "orphans: "
```
Comment (ojmarcelino) [inline po/pt_BR.po:198]: Just a tiny suggestion, orphans as _órfãos_ in pt-br. Not sure if worth open a review just for this...

**Pull request merged**